packages/react-dom/src/__tests__/ReactBrowserEventEmitter-test.internal.js

```diff
@@ -9,360 +9,480 @@
 
 'use strict';
 
-let EventPluginHub;
-let EventPluginRegistry;
 let React;
 let ReactDOM;
-let ReactDOMComponentTree;
-let ReactBrowserEventEmitter;
 let ReactTestUtils;
-
-let idCallOrder;
-const recordID = function(id) {
-  idCallOrder.push(id);
-};
-const recordIDAndStopPropagation = function(id, event) {
-  recordID(id);
-  event.stopPropagation();
-};
-const recordIDAndReturnFalse = function(id, event) {
-  recordID(id);
-  return false;
-};
-const LISTENER = jest.fn();
-const ON_CLICK_KEY = 'onClick';
-const ON_CHANGE_KEY = 'onChange';
-const ON_MOUSE_ENTER_KEY = 'onMouseEnter';
-
-let GRANDPARENT;
-let PARENT;
-let CHILD;
-
-let getListener;
-let putListener;
-let deleteAllListeners;
+let LISTENER;
 
 let container;
 
-function registerSimpleTestHandler() {
-  putListener(CHILD, ON_CLICK_KEY, LISTENER);
-  const listener = getListener(CHILD, ON_CLICK_KEY);
-  expect(listener).toEqual(LISTENER);
-  return getListener(CHILD, ON_CLICK_KEY);
-}
+let createInitialInstance;
+
+let updateInstance;
+
+let childRef;
 
 describe('ReactBrowserEventEmitter', () => {
   beforeEach(() => {
     jest.resetModules();
-    LISTENER.mockClear();
+    LISTENER = jest.fn();
```

Comment:
I think we don't need to have `LISTENER` as an variable in the closure and can just call `jest.fn()` whenever we want a listener :-)